src/Routing/UrlGenerator.php

```diff
@@ -43,6 +45,10 @@ class UrlGenerator
      */
     public function __construct(Application $app)
     {
+        if (! $app instanceof Lumen) {
```

Comment:
Okay, I just add it as fallback, but I don't really think it's required.

So I need to fix this and send to master branch?